packages/react-theme-provider/src/components/ThemeProvider/ThemeProvider.types.ts

```diff
@@ -4,6 +4,7 @@ import { PartialTheme, Theme } from '@fluentui/react-theme';
 
 export interface ThemeProviderProps extends ComponentProps, React.HTMLAttributes<HTMLElement> {
   theme?: PartialTheme | Theme;
+  targetDocument?: Document | undefined;
```

Comment:
could you add comments on what the intent is for this?

Reply:
Added comment